targetplatform/smarthome.target

```diff
@@ -153,6 +153,7 @@
 <unit id="org.eclipse.jetty.deploy" version="9.2.9.v20150224"/>
 <unit id="org.eclipse.jetty.osgi.boot" version="9.2.9.v20150224"/>
 <unit id="org.eclipse.jetty.osgi.httpservice" version="9.2.9.v20150224"/>
+<unit id="org.eclipse.jetty.proxy" version="9.2.9.v20150224"/>
```

Comment:
What is the correct way to add this JAR to the target platform?  The PR build failed with:

```
[ERROR] Failed to execute goal org.eclipse.tycho:tycho-compiler-plugin:0.23.1:compile (default-compile) on project org.eclipse.smarthome.ui: Compilation failure: Compilation failure:
[ERROR] /home/travis/build/eclipse/smarthome/bundles/ui/org.eclipse.smarthome.ui/src/main/java/org/eclipse/smarthome/ui/internal/proxy/ProxyServlet.java:[176]
[ERROR] protected void copyRequestHeaders(HttpServletRequest clientRequest, Request proxyRequest) {
[ERROR] ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[ERROR] The method copyRequestHeaders(HttpServletRequest, Request) of type ProxyServlet must override or implement a supertype method
[ERROR] /home/travis/build/eclipse/smarthome/bundles/ui/org.eclipse.smarthome.ui/src/main/java/org/eclipse/smarthome/ui/internal/proxy/ProxyServlet.java:[177]
[ERROR] super.copyRequestHeaders(clientRequest, proxyRequest);
[ERROR] ^^^^^^^^^^^^^^^^^^
[ERROR] The method copyRequestHeaders(HttpServletRequest, Request) is undefined for the type AsyncProxyServlet
[ERROR] 2 problems (2 errors)
```

Many thanks for guidance.